SQL Server - `Incorrect syntax near the keyword 'null'. [Failed SQL: (156) USE null]`

## Environment

**Liquibase Version**: 4.15 (started in 4.13)

**Liquibase Integration & Version**: CLI

**Liquibase Extension(s) & Version**: None

**Database Vendor & Version**: SQL Server (Azure SQL and local SQL Server)

**Operating System Type & Version**: Windows (also reproducible on Linux)

**Infrastructure Type/Provider**: Azure (also reproducible on local)


## Description

The PR #2940 introduces a breaking change for SQL Server when `defaultSchemaName` is included without a `defaultCatalogName`. This command:

```
liquibase.bat --driver="com.microsoft.sqlserver.jdbc.SQLServerDriver" --url=jdbc:"sqlserver://host.docker.internal;databaseName=REDACTED;trustServerCertificate=true;" --defaultSchemaName="dbo" --username="REDACTED" --password="REDACTED" --changeLogFile="REDACTED" --logLevel="debug" --logFile="REDACTED" --defaultsFile="liquibase.properties" update
```

produces this error:

```
[2022-08-17 15:13:31] SEVERE [liquibase.integration] Incorrect syntax near the keyword 'null'. [Failed SQL: (156) USE null]
liquibase.exception.CommandExecutionException: liquibase.exception.LiquibaseException: Unexpected error running Liquibase: Incorrect syntax near the keyword 'null'. [Failed SQL: (156) USE null]
	at liquibase.command.CommandScope.execute(CommandScope.java:162)
	at liquibase.integration.commandline.CommandRunner.call(CommandRunner.java:51)
	at liquibase.integration.commandline.CommandRunner.call(CommandRunner.java:21)
	at picocli.CommandLine.executeUserObject(CommandLine.java:1953)
	at picocli.CommandLine.access$1300(CommandLine.java:145)
	at picocli.CommandLine$RunLast.executeUserObjectOfLastSubcommandWithSameParent(CommandLine.java:2358)
	at picocli.CommandLine$RunLast.handle(CommandLine.java:2352)
	at picocli.CommandLine$RunLast.handle(CommandLine.java:2314)
	at picocli.CommandLine$AbstractParseResultHandler.execute(CommandLine.java:2179)
	at picocli.CommandLine$RunLast.execute(CommandLine.java:2316)
	at picocli.CommandLine.execute(CommandLine.java:2078)
	at liquibase.integration.commandline.LiquibaseCommandLine.lambda$execute$1(LiquibaseCommandLine.java:334)
	at liquibase.Scope.child(Scope.java:189)
	at liquibase.Scope.child(Scope.java:165)
	at liquibase.integration.commandline.LiquibaseCommandLine.execute(LiquibaseCommandLine.java:299)
	at liquibase.integration.commandline.LiquibaseCommandLine.main(LiquibaseCommandLine.java:84)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(Unknown Source)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(Unknown Source)
	at java.lang.reflect.Method.invoke(Unknown Source)
	at liquibase.integration.commandline.LiquibaseLauncher.main(LiquibaseLauncher.java:91)
Caused by: liquibase.exception.LiquibaseException: Unexpected error running Liquibase: Incorrect syntax near the keyword 'null'. [Failed SQL: (156) USE null]
	at liquibase.integration.commandline.Main$1.run(Main.java:440)
	at liquibase.integration.commandline.Main$1.run(Main.java:218)
	at liquibase.Scope.child(Scope.java:189)
	at liquibase.Scope.child(Scope.java:165)
	at liquibase.integration.commandline.Main.run(Main.java:218)
	at liquibase.command.AbstractCliWrapperCommandStep.run(AbstractCliWrapperCommandStep.java:33)
	at liquibase.command.CommandScope.execute(CommandScope.java:156)
	... 20 more
Caused by: liquibase.exception.DatabaseException: liquibase.exception.DatabaseException: Incorrect syntax near the keyword 'null'. [Failed SQL: (156) USE null]
	at liquibase.integration.commandline.CommandLineUtils.createDatabaseObject(CommandLineUtils.java:138)
	at liquibase.integration.commandline.Main.doMigration(Main.java:1479)
	at liquibase.integration.commandline.Main$1.lambda$run$0(Main.java:395)
	at liquibase.Scope.lambda$child$0(Scope.java:180)
	at liquibase.Scope.child(Scope.java:189)
	at liquibase.Scope.child(Scope.java:179)
	at liquibase.Scope.child(Scope.java:158)
	at liquibase.integration.commandline.Main$1.run(Main.java:394)
	... 26 more
Caused by: liquibase.exception.DatabaseException: Incorrect syntax near the keyword 'null'. [Failed SQL: (156) USE null]
	at liquibase.executor.jvm.JdbcExecutor$ExecuteStatementCallback.doInStatement(JdbcExecutor.java:434)
	at liquibase.executor.jvm.JdbcExecutor.execute(JdbcExecutor.java:77)
	at liquibase.executor.jvm.JdbcExecutor.execute(JdbcExecutor.java:160)
	at liquibase.executor.jvm.JdbcExecutor.execute(JdbcExecutor.java:125)
	at liquibase.database.core.DatabaseUtils.initializeDatabase(DatabaseUtils.java:63)
	at liquibase.integration.commandline.CommandLineUtils.createDatabaseObject(CommandLineUtils.java:134)
	... 33 more
Caused by: com.microsoft.sqlserver.jdbc.SQLServerException: Incorrect syntax near the keyword 'null'.
	at com.microsoft.sqlserver.jdbc.SQLServerException.makeFromDatabaseError(SQLServerException.java:265)
	at com.microsoft.sqlserver.jdbc.SQLServerStatement.getNextResult(SQLServerStatement.java:1676)
	at com.microsoft.sqlserver.jdbc.SQLServerStatement.doExecuteStatement(SQLServerStatement.java:907)
	at com.microsoft.sqlserver.jdbc.SQLServerStatement$StmtExecCmd.doExecute(SQLServerStatement.java:802)
	at com.microsoft.sqlserver.jdbc.TDSCommand.execute(IOBuffer.java:7730)
	at com.microsoft.sqlserver.jdbc.SQLServerConnection.executeCommand(SQLServerConnection.java:3786)
	at com.microsoft.sqlserver.jdbc.SQLServerStatement.executeCommand(SQLServerStatement.java:268)
	at com.microsoft.sqlserver.jdbc.SQLServerStatement.executeStatement(SQLServerStatement.java:242)
	at com.microsoft.sqlserver.jdbc.SQLServerStatement.execute(SQLServerStatement.java:775)
	at liquibase.executor.jvm.JdbcExecutor$ExecuteStatementCallback.doInStatement(JdbcExecutor.java:430)
	... 38 more
```

We have determined that we can either leave off `defaultSchemaName` or add `defaultCatalogName` (but we need to properly escape it which seems wrong, like: `--defaultCatalogName "[MyDatabase-That-Has-Hyphens]"`).

Either way, I think the bug with this change is all the other database types use schema name and catalog name somewhat interchangeably, which is not true with SQL Server. The `USE` statement should not be run for SQL Server if `defaultCatalogName` is not specified.

## Steps To Reproduce

The only important parts of the following command seem to be:
1. Include `defaultSchemaName`
2. Do not include `defaultCatalogName`

```
liquibase.bat --driver="com.microsoft.sqlserver.jdbc.SQLServerDriver" --url=jdbc:"sqlserver://host.docker.internal;databaseName=REDACTED;trustServerCertificate=true;" --defaultSchemaName="dbo" --username="REDACTED" --password="REDACTED" --changeLogFile="REDACTED" --logLevel="debug" --logFile="REDACTED" --defaultsFile="liquibase.properties" update
```

## Actual Behavior

An exception is thrown.

## Expected/Desired Behavior

We expect Liquibase to work like it did previously and run the `update` without crashing.

## Screenshots (if appropriate)
N/A

## Additional Context

Originally reported on PR #2940

Also ran with the Liquibase docker container for demonstration:

```
 docker run --rm -it liquibase/liquibase:4.15 --defaultSchemaName dbo --driver="com.microsoft.sqlserver.jdbc.SQLServerDriver" --url=jdbc:"sqlserver://host.docker.internal;databaseName=REDACTED;trustServerCertificate=true;" --username="sa" --password="REDACTED"
 --changeLogFile="doesNotExist.xml" --logLevel="FINE" update
```

<details>
  <summary>Exception Details</summary>
```
[2022-08-18 16:41:26] FINE [liquibase.resource] Adding path /liquibase/. to resourceAccessor liquibase.resource.FileSystemResourceAccessor
[2022-08-18 16:41:26] FINE [liquibase.configuration] No configuration value for liquibase.searchPath found
[2022-08-18 16:41:26] FINE [liquibase.resource] Overall search path:
  - /liquibase/.
  - /liquibase/changelog
  - /liquibase/classpath
  - /liquibase/internal/lib
  - /liquibase/internal/lib/commons-collections4.jar
  - /liquibase/internal/lib/commons-lang3.jar
  - /liquibase/internal/lib/commons-text.jar
  - /liquibase/internal/lib/connector-api.jar
  - /liquibase/internal/lib/h2.jar
  - /liquibase/internal/lib/hsqldb.jar
  - /liquibase/internal/lib/jaxb-api.jar
  - /liquibase/internal/lib/jaxb-core.jar
  - /liquibase/internal/lib/jaxb-runtime.jar
  - /liquibase/internal/lib/jaybird.jar
  - /liquibase/internal/lib/jcc.jar
  - /liquibase/internal/lib/liquibase-commercial.jar
  - /liquibase/internal/lib/liquibase-core.jar
  - /liquibase/internal/lib/mariadb-java-client.jar
  - /liquibase/internal/lib/mssql-jdbc.jar
  - /liquibase/internal/lib/ojdbc8.jar
  - /liquibase/internal/lib/opencsv.jar
  - /liquibase/internal/lib/picocli.jar
  - /liquibase/internal/lib/postgresql.jar
  - /liquibase/internal/lib/snakeyaml.jar
  - /liquibase/internal/lib/snowflake-jdbc.jar
  - /liquibase/internal/lib/sqlite-jdbc.jar
  - /liquibase/lib
[2022-08-18 16:41:26] FINE [liquibase.configuration] No configuration value for liquibase.shouldRun aka should.run found
[2022-08-18 16:41:26] FINE [liquibase.configuration] Configuration liquibase.shouldRun is using the default value of true
[2022-08-18 16:41:26] FINE [liquibase.configuration] No configuration value for liquibase.showBanner found
[2022-08-18 16:41:26] FINE [liquibase.configuration] Configuration liquibase.showBanner is using the default value of true
####################################################
##   _     _             _ _                      ##
##  | |   (_)           (_) |                     ##
##  | |    _  __ _ _   _ _| |__   __ _ ___  ___   ##
##  | |   | |/ _` | | | | | '_ \ / _` / __|/ _ \  ##
##  | |___| | (_| | |_| | | |_) | (_| \__ \  __/  ##
##  \_____/_|\__, |\__,_|_|_.__/ \__,_|___/\___|  ##
##              | |                               ##
##              |_|                               ##
##                                                ##
##  Get documentation at docs.liquibase.com       ##
##  Get certified courses at learn.liquibase.com  ##
##  Free schema change activity reports at        ##
##      https://hub.liquibase.com                 ##
##                                                ##
####################################################
Starting Liquibase at 16:41:26 (version 4.15.0 #4001 built at 2022-08-05 16:17+0000)
Liquibase Version: 4.15.0
Liquibase Community 4.15.0 by Liquibase
[2022-08-18 16:41:26] FINE [liquibase.configuration] No configuration value for liquibase.monitorPerformance found
[2022-08-18 16:41:26] FINE [liquibase.configuration] Configuration liquibase.monitorPerformance is using the default value of false
[2022-08-18 16:41:26] FINE [liquibase.integration] Performance monitoring disabled
[2022-08-18 16:41:26] FINE [liquibase.configuration] No configuration value for liquibase.outputFile found
[2022-08-18 16:41:26] FINE [liquibase.configuration] No configuration value for liquibase.strict found
[2022-08-18 16:41:26] FINE [liquibase.configuration] Configuration liquibase.strict is using the default value of false
[2022-08-18 16:41:26] FINE [liquibase.configuration] No configuration value for liquibase.command.update.changeExecListenerClass found
[2022-08-18 16:41:26] FINE [liquibase.configuration] No configuration value for liquibase.command.changeExecListenerClass found
[2022-08-18 16:41:26] FINE [liquibase.configuration] No configuration value for liquibase.command.update.changeExecListenerPropertiesFile found
[2022-08-18 16:41:26] FINE [liquibase.configuration] No configuration value for liquibase.command.changeExecListenerPropertiesFile found
[2022-08-18 16:41:26] FINE [liquibase.configuration] No configuration value for liquibase.command.update.changelogFile found
[2022-08-18 16:41:26] FINE [liquibase.configuration] Found 'liquibase.command.changelogFile' configuration of 'doesNotExist.xml'
    command argument '--changelogFile' of 'doesNotExist.xml'
[2022-08-18 16:41:26] FINE [liquibase.configuration] No configuration value for liquibase.command.update.contexts found
[2022-08-18 16:41:26] FINE [liquibase.configuration] No configuration value for liquibase.command.contexts found
[2022-08-18 16:41:26] FINE [liquibase.configuration] No configuration value for liquibase.command.update.defaultCatalogName found
[2022-08-18 16:41:26] FINE [liquibase.configuration] No configuration value for liquibase.command.defaultCatalogName found
[2022-08-18 16:41:26] FINE [liquibase.configuration] No configuration value for liquibase.command.update.defaultSchemaName found
[2022-08-18 16:41:26] FINE [liquibase.configuration] Found 'liquibase.command.defaultSchemaName' configuration of 'dbo'
    command argument '--defaultSchemaName' of 'dbo'
[2022-08-18 16:41:26] FINE [liquibase.configuration] No configuration value for liquibase.command.update.driver found
[2022-08-18 16:41:26] FINE [liquibase.configuration] Found 'liquibase.command.driver' configuration of 'com.microsoft.sqlserver.jdbc.SQLServerDriver'
    command argument '--driver' of 'com.microsoft.sqlserver.jdbc.SQLServerDriver'
[2022-08-18 16:41:26] FINE [liquibase.configuration] No configuration value for liquibase.command.update.driverPropertiesFile found
[2022-08-18 16:41:26] FINE [liquibase.configuration] No configuration value for liquibase.command.driverPropertiesFile found
[2022-08-18 16:41:26] FINE [liquibase.configuration] No configuration value for liquibase.command.update.labels found
[2022-08-18 16:41:26] FINE [liquibase.configuration] No configuration value for liquibase.command.labels found
[2022-08-18 16:41:26] FINE [liquibase.configuration] No configuration value for liquibase.command.update.password found
[2022-08-18 16:41:26] FINE [liquibase.configuration] Found 'liquibase.command.password' configuration of '*****'
    command argument '--password' of '*****'
[2022-08-18 16:41:26] FINE [liquibase.configuration] No configuration value for liquibase.command.update.url found
[2022-08-18 16:41:26] FINE [liquibase.configuration] Found 'liquibase.command.url' configuration of 'jdbc:sqlserver://host.docker.internal;databaseName=REDACTED;trustServerCertificate=true;'
    command argument '--url' of 'jdbc:sqlserver://host.docker.internal;databaseName=REDACTED;trustServerCertificate=true;'
[2022-08-18 16:41:26] FINE [liquibase.configuration] No configuration value for liquibase.command.update.username found
[2022-08-18 16:41:26] FINE [liquibase.configuration] Found 'liquibase.command.username' configuration of 'sa'
    command argument '--username' of 'sa'
[2022-08-18 16:41:26] FINE [liquibase.command] Pipeline for command 'update: liquibase.command.core.UpdateCommandStep
[2022-08-18 16:41:26] FINE [liquibase.configuration] No configuration value for liquibase.hub.apiKey found
[2022-08-18 16:41:26] FINE [liquibase.configuration] No configuration value for liquibase.propertyProviderClass found
[2022-08-18 16:41:26] FINE [liquibase.configuration] No configuration value for liquibase.databaseClass found
[2022-08-18 16:41:26] FINE [liquibase.configuration] No configuration value for liquibase.hub.url found
[2022-08-18 16:41:26] FINE [liquibase.configuration] Configuration liquibase.hub.url is using the default value of https://hub.liquibase.com
[2022-08-18 16:41:26] FINE [liquibase.integration] Liquibase Hub URL:      https://hub.liquibase.com
[2022-08-18 16:41:26] FINE [liquibase.configuration] No configuration value for liquibase.hub.mode found
[2022-08-18 16:41:26] FINE [liquibase.configuration] Configuration liquibase.hub.mode is using the default value of ALL
[2022-08-18 16:41:26] FINE [liquibase.integration] Liquibase Hub Mode:     ALL
[2022-08-18 16:41:26] FINE [liquibase.servicelocator] Loaded liquibase.database.Database instance liquibase.database.core.CockroachDatabase
[2022-08-18 16:41:26] FINE [liquibase.servicelocator] Loaded liquibase.database.Database instance liquibase.database.core.DB2Database
[2022-08-18 16:41:26] FINE [liquibase.servicelocator] Loaded liquibase.database.Database instance liquibase.database.core.Db2zDatabase
[2022-08-18 16:41:26] FINE [liquibase.servicelocator] Loaded liquibase.database.Database instance liquibase.database.core.DerbyDatabase
[2022-08-18 16:41:26] FINE [liquibase.servicelocator] Loaded liquibase.database.Database instance liquibase.database.core.EnterpriseDBDatabase
[2022-08-18 16:41:26] FINE [liquibase.servicelocator] Loaded liquibase.database.Database instance liquibase.database.core.FirebirdDatabase
[2022-08-18 16:41:26] FINE [liquibase.servicelocator] Loaded liquibase.database.Database instance liquibase.database.core.H2Database
[2022-08-18 16:41:26] FINE [liquibase.servicelocator] Loaded liquibase.database.Database instance liquibase.database.core.HsqlDatabase
[2022-08-18 16:41:26] FINE [liquibase.servicelocator] Loaded liquibase.database.Database instance liquibase.database.core.InformixDatabase
[2022-08-18 16:41:26] FINE [liquibase.servicelocator] Loaded liquibase.database.Database instance liquibase.database.core.Ingres9Database
[2022-08-18 16:41:26] FINE [liquibase.servicelocator] Loaded liquibase.database.Database instance liquibase.database.core.MSSQLDatabase
[2022-08-18 16:41:26] FINE [liquibase.servicelocator] Loaded liquibase.database.Database instance liquibase.database.core.MariaDBDatabase
[2022-08-18 16:41:26] FINE [liquibase.servicelocator] Loaded liquibase.database.Database instance liquibase.database.core.MockDatabase
[2022-08-18 16:41:26] FINE [liquibase.servicelocator] Loaded liquibase.database.Database instance liquibase.database.core.MySQLDatabase
[2022-08-18 16:41:26] FINE [liquibase.servicelocator] Loaded liquibase.database.Database instance liquibase.database.core.OracleDatabase
[2022-08-18 16:41:26] FINE [liquibase.servicelocator] Loaded liquibase.database.Database instance liquibase.database.core.PostgresDatabase
[2022-08-18 16:41:26] FINE [liquibase.servicelocator] Loaded liquibase.database.Database instance liquibase.database.core.SQLiteDatabase
[2022-08-18 16:41:26] FINE [liquibase.servicelocator] Loaded liquibase.database.Database instance liquibase.database.core.SybaseASADatabase
[2022-08-18 16:41:26] FINE [liquibase.servicelocator] Loaded liquibase.database.Database instance liquibase.database.core.SybaseDatabase
[2022-08-18 16:41:26] FINE [liquibase.servicelocator] Loaded liquibase.database.Database instance liquibase.database.core.UnsupportedDatabase
[2022-08-18 16:41:26] FINE [liquibase.servicelocator] Loaded liquibase.database.Database instance liquibase.database.core.SnowflakeDatabase
[2022-08-18 16:41:26] FINE [liquibase.database] Properties:
[2022-08-18 16:41:26] FINE [liquibase.database] Key:'password' Value:'**********'
[2022-08-18 16:41:26] FINE [liquibase.database] Key:'user' Value:'sa'
[2022-08-18 16:41:26] FINE [liquibase.database] Connecting to the URL:'jdbc:sqlserver://host.docker.internal;databaseName=REDACTED;trustServerCertificate=true;' using driver:'com.microsoft.sqlserver.jdbc.SQLServerDriver'
[2022-08-18 16:41:26] FINE [liquibase.servicelocator] Loaded liquibase.database.DatabaseConnection instance liquibase.database.jvm.JdbcConnection
[2022-08-18 16:41:26] FINE [liquibase.servicelocator] Loaded liquibase.database.DatabaseConnection instance com.datical.liquibase.ext.database.jvm.ProJdbcConnection
[2022-08-18 16:41:26] FINE [liquibase.database] Connection has been created
[2022-08-18 16:41:26] FINE [liquibase.database] Connected to sa@jdbc:sqlserver://host.docker.internal:1433;connectRetryInterval=10;connectRetryCount=1;maxResultBuffer=-1;sendTemporalDataTypesAsStringForBulkCopy=true;delayLoadingLobs=true;useFmtOnly=false;useBulkCopyForBatchInsert=false;cancelQueryTimeout=-1;sslProtocol=TLS;jaasConfigurationName=SQLJDBCDriver;statementPoolingCacheSize=0;serverPreparedStatementDiscardThreshold=10;enablePrepareOnFirstPreparedStatementCall=false;fips=false;socketTimeout=0;authentication=NotSpecified;authenticationScheme=nativeAuthentication;xopenStates=false;sendTimeAsDatetime=true;replication=false;trustStoreType=JKS;trustServerCertificate=true;TransparentNetworkIPResolution=true;serverNameAsACE=false;sendStringParametersAsUnicode=true;selectMethod=direct;responseBuffering=adaptive;queryTimeout=-1;packetSize=8000;multiSubnetFailover=false;loginTimeout=15;lockTimeout=-1;lastUpdateCount=true;encrypt=true;disableStatementPooling=true;databaseName=REDACTED;columnEncryptionSetting=Disabled;applicationName=Microsoft JDBC Driver for SQL Server;applicationIntent=readwrite;
[2022-08-18 16:41:26] FINE [liquibase.database] Setting auto commit to false from true
[2022-08-18 16:41:26] FINE [liquibase.configuration] No configuration value for liquibase.preserveSchemaCase found
[2022-08-18 16:41:26] FINE [liquibase.configuration] Configuration liquibase.preserveSchemaCase is using the default value of false
[2022-08-18 16:41:26] FINE [liquibase.servicelocator] Loaded liquibase.executor.Executor instance liquibase.executor.jvm.JdbcExecutor
[2022-08-18 16:41:26] FINE [liquibase.servicelocator] Loaded liquibase.executor.Executor instance com.datical.liquibase.ext.tools.OracleSqlPlusExecutor
[2022-08-18 16:41:26] FINE [liquibase.servicelocator] Loaded liquibase.executor.Executor instance com.datical.liquibase.ext.tools.MssqlSqlcmdExecutor
[2022-08-18 16:41:26] FINE [liquibase.servicelocator] Loaded liquibase.executor.Executor instance com.datical.liquibase.ext.tools.PsqlExecutor
[2022-08-18 16:41:26] FINE [liquibase.executor] Executing with the 'jdbc' executor
[2022-08-18 16:41:26] FINE [liquibase.servicelocator] Loaded liquibase.sqlgenerator.SqlGenerator instance liquibase.sqlgenerator.core.AddAutoIncrementGenerator
[2022-08-18 16:41:26] FINE [liquibase.servicelocator] Loaded liquibase.sqlgenerator.SqlGenerator instance liquibase.sqlgenerator.core.AddAutoIncrementGeneratorDB2
[2022-08-18 16:41:26] FINE [liquibase.servicelocator] Loaded liquibase.sqlgenerator.SqlGenerator instance liquibase.sqlgenerator.core.AddAutoIncrementGeneratorHsqlH2
[2022-08-18 16:41:26] FINE [liquibase.servicelocator] Loaded liquibase.sqlgenerator.SqlGenerator instance liquibase.sqlgenerator.core.AddAutoIncrementGeneratorInformix
[2022-08-18 16:41:26] FINE [liquibase.servicelocator] Loaded liquibase.sqlgenerator.SqlGenerator instance liquibase.sqlgenerator.core.AddAutoIncrementGeneratorMySQL
[2022-08-18 16:41:26] FINE [liquibase.servicelocator] Loaded liquibase.sqlgenerator.SqlGenerator instance liquibase.sqlgenerator.core.AddAutoIncrementGeneratorSQLite
[2022-08-18 16:41:26] FINE [liquibase.servicelocator] Loaded liquibase.sqlgenerator.SqlGenerator instance liquibase.sqlgenerator.core.AddColumnGenerator
[2022-08-18 16:41:26] FINE [liquibase.servicelocator] Loaded liquibase.sqlgenerator.SqlGenerator instance liquibase.sqlgenerator.core.AddColumnGeneratorDefaultClauseBeforeNotNull
[2022-08-18 16:41:26] FINE [liquibase.servicelocator] Loaded liquibase.sqlgenerator.SqlGenerator instance liquibase.sqlgenerator.core.AddColumnGeneratorSQLite
[2022-08-18 16:41:26] FINE [liquibase.servicelocator] Loaded liquibase.sqlgenerator.SqlGenerator instance liquibase.sqlgenerator.core.AddDefaultValueGenerator
[2022-08-18 16:41:26] FINE [liquibase.servicelocator] Loaded liquibase.sqlgenerator.SqlGenerator instance liquibase.sqlgenerator.core.AddDefaultValueGeneratorDerby
[2022-08-18 16:41:26] FINE [liquibase.servicelocator] Loaded liquibase.sqlgenerator.SqlGenerator instance liquibase.sqlgenerator.core.AddDefaultValueGeneratorInformix
[2022-08-18 16:41:26] FINE [liquibase.servicelocator] Loaded liquibase.sqlgenerator.SqlGenerator instance liquibase.sqlgenerator.core.AddDefaultValueGeneratorMSSQL
[2022-08-18 16:41:26] FINE [liquibase.servicelocator] Loaded liquibase.sqlgenerator.SqlGenerator instance liquibase.sqlgenerator.core.AddDefaultValueGeneratorMySQL
[2022-08-18 16:41:26] FINE [liquibase.servicelocator] Loaded liquibase.sqlgenerator.SqlGenerator instance liquibase.sqlgenerator.core.AddDefaultValueGeneratorOracle
[2022-08-18 16:41:26] FINE [liquibase.servicelocator] Loaded liquibase.sqlgenerator.SqlGenerator instance liquibase.sqlgenerator.core.AddDefaultValueGeneratorPostgres
[2022-08-18 16:41:26] FINE [liquibase.servicelocator] Loaded liquibase.sqlgenerator.SqlGenerator instance liquibase.sqlgenerator.core.AddDefaultValueGeneratorSQLite
[2022-08-18 16:41:26] FINE [liquibase.servicelocator] Loaded liquibase.sqlgenerator.SqlGenerator instance liquibase.sqlgenerator.core.AddDefaultValueGeneratorSybase
[2022-08-18 16:41:26] FINE [liquibase.servicelocator] Loaded liquibase.sqlgenerator.SqlGenerator instance liquibase.sqlgenerator.core.AddDefaultValueGeneratorSybaseASA
[2022-08-18 16:41:26] FINE [liquibase.servicelocator] Loaded liquibase.sqlgenerator.SqlGenerator instance liquibase.sqlgenerator.core.AddForeignKeyConstraintGenerator
[2022-08-18 16:41:26] FINE [liquibase.servicelocator] Loaded liquibase.sqlgenerator.SqlGenerator instance liquibase.sqlgenerator.core.AddPrimaryKeyGenerator
[2022-08-18 16:41:26] FINE [liquibase.servicelocator] Loaded liquibase.sqlgenerator.SqlGenerator instance liquibase.sqlgenerator.core.AddPrimaryKeyGeneratorInformix
[2022-08-18 16:41:26] FINE [liquibase.servicelocator] Loaded liquibase.sqlgenerator.SqlGenerator instance liquibase.sqlgenerator.core.AddUniqueConstraintGenerator
[2022-08-18 16:41:26] FINE [liquibase.servicelocator] Loaded liquibase.sqlgenerator.SqlGenerator instance liquibase.sqlgenerator.core.AddUniqueConstraintGeneratorInformix
[2022-08-18 16:41:26] FINE [liquibase.servicelocator] Loaded liquibase.sqlgenerator.SqlGenerator instance liquibase.sqlgenerator.core.AddUniqueConstraintGeneratorTDS
[2022-08-18 16:41:26] FINE [liquibase.servicelocator] Loaded liquibase.sqlgenerator.SqlGenerator instance liquibase.sqlgenerator.core.AlterSequenceGenerator
[2022-08-18 16:41:26] FINE [liquibase.servicelocator] Loaded liquibase.sqlgenerator.SqlGenerator instance liquibase.sqlgenerator.core.BatchDmlExecutablePreparedStatementGenerator
[2022-08-18 16:41:26] FINE [liquibase.servicelocator] Loaded liquibase.sqlgenerator.SqlGenerator instance liquibase.sqlgenerator.core.ClearDatabaseChangeLogTableGenerator
[2022-08-18 16:41:26] FINE [liquibase.servicelocator] Loaded liquibase.sqlgenerator.SqlGenerator instance liquibase.sqlgenerator.core.CommentGenerator
[2022-08-18 16:41:26] FINE [liquibase.servicelocator] Loaded liquibase.sqlgenerator.SqlGenerator instance liquibase.sqlgenerator.core.CopyRowsGenerator
[2022-08-18 16:41:26] FINE [liquibase.servicelocator] Loaded liquibase.sqlgenerator.SqlGenerator instance liquibase.sqlgenerator.core.CreateDatabaseChangeLogLockTableGenerator
[2022-08-18 16:41:26] FINE [liquibase.servicelocator] Loaded liquibase.sqlgenerator.SqlGenerator instance liquibase.sqlgenerator.core.CreateDatabaseChangeLogTableGenerator
[2022-08-18 16:41:26] FINE [liquibase.servicelocator] Loaded liquibase.sqlgenerator.SqlGenerator instance liquibase.sqlgenerator.core.CreateDatabaseChangeLogTableGeneratorSybase
[2022-08-18 16:41:26] FINE [liquibase.servicelocator] Loaded liquibase.sqlgenerator.SqlGenerator instance liquibase.sqlgenerator.core.CreateIndexGenerator
[2022-08-18 16:41:26] FINE [liquibase.servicelocator] Loaded liquibase.sqlgenerator.SqlGenerator instance liquibase.sqlgenerator.core.CreateIndexGeneratorFirebird
[2022-08-18 16:41:26] FINE [liquibase.servicelocator] Loaded liquibase.sqlgenerator.SqlGenerator instance liquibase.sqlgenerator.core.CreateIndexGeneratorPostgres
[2022-08-18 16:41:26] FINE [liquibase.servicelocator] Loaded liquibase.sqlgenerator.SqlGenerator instance liquibase.sqlgenerator.core.CreateProcedureGenerator
[2022-08-18 16:41:26] FINE [liquibase.servicelocator] Loaded liquibase.sqlgenerator.SqlGenerator instance liquibase.sqlgenerator.core.CreateSequenceGenerator
[2022-08-18 16:41:26] FINE [liquibase.servicelocator] Loaded liquibase.sqlgenerator.SqlGenerator instance liquibase.sqlgenerator.core.CreateTableGenerator
[2022-08-18 16:41:26] FINE [liquibase.servicelocator] Loaded liquibase.sqlgenerator.SqlGenerator instance liquibase.sqlgenerator.core.CreateTableGeneratorInformix
[2022-08-18 16:41:26] FINE [liquibase.servicelocator] Loaded liquibase.sqlgenerator.SqlGenerator instance liquibase.sqlgenerator.core.CreateViewGenerator
[2022-08-18 16:41:26] FINE [liquibase.servicelocator] Loaded liquibase.sqlgenerator.SqlGenerator instance liquibase.sqlgenerator.core.CreateViewGeneratorInformix
[2022-08-18 16:41:26] FINE [liquibase.servicelocator] Loaded liquibase.sqlgenerator.SqlGenerator instance liquibase.sqlgenerator.core.DeleteGenerator
[2022-08-18 16:41:26] FINE [liquibase.servicelocator] Loaded liquibase.sqlgenerator.SqlGenerator instance liquibase.sqlgenerator.core.DropColumnGenerator
[2022-08-18 16:41:26] FINE [liquibase.servicelocator] Loaded liquibase.sqlgenerator.SqlGenerator instance liquibase.sqlgenerator.core.DropDefaultValueGenerator
[2022-08-18 16:41:26] FINE [liquibase.servicelocator] Loaded liquibase.sqlgenerator.SqlGenerator instance liquibase.sqlgenerator.core.DropForeignKeyConstraintGenerator
[2022-08-18 16:41:26] FINE [liquibase.servicelocator] Loaded liquibase.sqlgenerator.SqlGenerator instance liquibase.sqlgenerator.core.DropIndexGenerator
[2022-08-18 16:41:26] FINE [liquibase.servicelocator] Loaded liquibase.sqlgenerator.SqlGenerator instance liquibase.sqlgenerator.core.DropPrimaryKeyGenerator
[2022-08-18 16:41:26] FINE [liquibase.servicelocator] Loaded liquibase.sqlgenerator.SqlGenerator instance liquibase.sqlgenerator.core.DropProcedureGenerator
[2022-08-18 16:41:26] FINE [liquibase.servicelocator] Loaded liquibase.sqlgenerator.SqlGenerator instance liquibase.sqlgenerator.core.DropSequenceGenerator
[2022-08-18 16:41:26] FINE [liquibase.servicelocator] Loaded liquibase.sqlgenerator.SqlGenerator instance liquibase.sqlgenerator.core.DropTableGenerator
[2022-08-18 16:41:26] FINE [liquibase.servicelocator] Loaded liquibase.sqlgenerator.SqlGenerator instance liquibase.sqlgenerator.core.DropUniqueConstraintGenerator
[2022-08-18 16:41:26] FINE [liquibase.servicelocator] Loaded liquibase.sqlgenerator.SqlGenerator instance liquibase.sqlgenerator.core.DropViewGenerator
[2022-08-18 16:41:26] FINE [liquibase.servicelocator] Loaded liquibase.sqlgenerator.SqlGenerator instance liquibase.sqlgenerator.core.GetNextChangeSetSequenceValueGenerator
[2022-08-18 16:41:26] FINE [liquibase.servicelocator] Loaded liquibase.sqlgenerator.SqlGenerator instance liquibase.sqlgenerator.core.GetViewDefinitionGenerator
[2022-08-18 16:41:26] FINE [liquibase.servicelocator] Loaded liquibase.sqlgenerator.SqlGenerator instance liquibase.sqlgenerator.core.GetViewDefinitionGeneratorDB2
[2022-08-18 16:41:26] FINE [liquibase.servicelocator] Loaded liquibase.sqlgenerator.SqlGenerator instance liquibase.sqlgenerator.core.GetViewDefinitionGeneratorDerby
[2022-08-18 16:41:26] FINE [liquibase.servicelocator] Loaded liquibase.sqlgenerator.SqlGenerator instance liquibase.sqlgenerator.core.GetViewDefinitionGeneratorFirebird
[2022-08-18 16:41:26] FINE [liquibase.servicelocator] Loaded liquibase.sqlgenerator.SqlGenerator instance liquibase.sqlgenerator.core.GetViewDefinitionGeneratorHsql
[2022-08-18 16:41:26] FINE [liquibase.servicelocator] Loaded liquibase.sqlgenerator.SqlGenerator instance liquibase.sqlgenerator.core.GetViewDefinitionGeneratorInformix
[2022-08-18 16:41:26] FINE [liquibase.servicelocator] Loaded liquibase.sqlgenerator.SqlGenerator instance liquibase.sqlgenerator.core.GetViewDefinitionGeneratorMSSQL
[2022-08-18 16:41:26] FINE [liquibase.servicelocator] Loaded liquibase.sqlgenerator.SqlGenerator instance liquibase.sqlgenerator.core.GetViewDefinitionGeneratorOracle
[2022-08-18 16:41:26] FINE [liquibase.servicelocator] Loaded liquibase.sqlgenerator.SqlGenerator instance liquibase.sqlgenerator.core.GetViewDefinitionGeneratorPostgres
[2022-08-18 16:41:26] FINE [liquibase.servicelocator] Loaded liquibase.sqlgenerator.SqlGenerator instance liquibase.sqlgenerator.core.GetViewDefinitionGeneratorSybase
[2022-08-18 16:41:26] FINE [liquibase.servicelocator] Loaded liquibase.sqlgenerator.SqlGenerator instance liquibase.sqlgenerator.core.GetViewDefinitionGeneratorSybaseASA
[2022-08-18 16:41:26] FINE [liquibase.servicelocator] Loaded liquibase.sqlgenerator.SqlGenerator instance liquibase.sqlgenerator.core.InitializeDatabaseChangeLogLockTableGenerator
[2022-08-18 16:41:26] FINE [liquibase.servicelocator] Loaded liquibase.sqlgenerator.SqlGenerator instance liquibase.sqlgenerator.core.InsertDataChangeGenerator
[2022-08-18 16:41:26] FINE [liquibase.servicelocator] Loaded liquibase.sqlgenerator.SqlGenerator instance liquibase.sqlgenerator.core.InsertGenerator
[2022-08-18 16:41:26] FINE [liquibase.servicelocator] Loaded liquibase.sqlgenerator.SqlGenerator instance liquibase.sqlgenerator.core.InsertOrUpdateGeneratorDB2
[2022-08-18 16:41:26] FINE [liquibase.servicelocator] Loaded liquibase.sqlgenerator.SqlGenerator instance liquibase.sqlgenerator.core.InsertOrUpdateGeneratorH2
[2022-08-18 16:41:26] FINE [liquibase.servicelocator] Loaded liquibase.sqlgenerator.SqlGenerator instance liquibase.sqlgenerator.core.InsertOrUpdateGeneratorHsql
[2022-08-18 16:41:26] FINE [liquibase.servicelocator] Loaded liquibase.sqlgenerator.SqlGenerator instance liquibase.sqlgenerator.core.InsertOrUpdateGeneratorInformix
[2022-08-18 16:41:26] FINE [liquibase.servicelocator] Loaded liquibase.sqlgenerator.SqlGenerator instance liquibase.sqlgenerator.core.InsertOrUpdateGeneratorMSSQL
[2022-08-18 16:41:26] FINE [liquibase.servicelocator] Loaded liquibase.sqlgenerator.SqlGenerator instance liquibase.sqlgenerator.core.InsertOrUpdateGeneratorMySQL
[2022-08-18 16:41:26] FINE [liquibase.servicelocator] Loaded liquibase.sqlgenerator.SqlGenerator instance liquibase.sqlgenerator.core.InsertOrUpdateGeneratorOracle
[2022-08-18 16:41:26] FINE [liquibase.servicelocator] Loaded liquibase.sqlgenerator.SqlGenerator instance liquibase.sqlgenerator.core.InsertOrUpdateGeneratorPostgres
[2022-08-18 16:41:26] FINE [liquibase.servicelocator] Loaded liquibase.sqlgenerator.SqlGenerator instance liquibase.sqlgenerator.core.InsertOrUpdateGeneratorSQLite
[2022-08-18 16:41:26] FINE [liquibase.servicelocator] Loaded liquibase.sqlgenerator.SqlGenerator instance liquibase.sqlgenerator.core.InsertOrUpdateGeneratorSybaseASA
[2022-08-18 16:41:26] FINE [liquibase.servicelocator] Loaded liquibase.sqlgenerator.SqlGenerator instance liquibase.sqlgenerator.core.InsertSetGenerator
[2022-08-18 16:41:26] FINE [liquibase.servicelocator] Loaded liquibase.sqlgenerator.SqlGenerator instance liquibase.sqlgenerator.core.LockDatabaseChangeLogGenerator
[2022-08-18 16:41:26] FINE [liquibase.servicelocator] Loaded liquibase.sqlgenerator.SqlGenerator instance liquibase.sqlgenerator.core.MarkChangeSetRanGenerator
[2022-08-18 16:41:26] FINE [liquibase.servicelocator] Loaded liquibase.sqlgenerator.SqlGenerator instance liquibase.sqlgenerator.core.ModifyDataTypeGenerator
[2022-08-18 16:41:26] FINE [liquibase.servicelocator] Loaded liquibase.sqlgenerator.SqlGenerator instance liquibase.sqlgenerator.core.RawSqlGenerator
[2022-08-18 16:41:26] FINE [liquibase.servicelocator] Loaded liquibase.sqlgenerator.SqlGenerator instance liquibase.sqlgenerator.core.ReindexGeneratorSQLite
[2022-08-18 16:41:26] FINE [liquibase.servicelocator] Loaded liquibase.sqlgenerator.SqlGenerator instance liquibase.sqlgenerator.core.RemoveChangeSetRanStatusGenerator
[2022-08-18 16:41:26] FINE [liquibase.servicelocator] Loaded liquibase.sqlgenerator.SqlGenerator instance liquibase.sqlgenerator.core.RenameColumnGenerator
[2022-08-18 16:41:26] FINE [liquibase.servicelocator] Loaded liquibase.sqlgenerator.SqlGenerator instance liquibase.sqlgenerator.core.RenameSequenceGenerator
[2022-08-18 16:41:26] FINE [liquibase.servicelocator] Loaded liquibase.sqlgenerator.SqlGenerator instance liquibase.sqlgenerator.core.RenameTableGenerator
[2022-08-18 16:41:26] FINE [liquibase.servicelocator] Loaded liquibase.sqlgenerator.SqlGenerator instance liquibase.sqlgenerator.core.RenameViewGenerator
[2022-08-18 16:41:26] FINE [liquibase.servicelocator] Loaded liquibase.sqlgenerator.SqlGenerator instance liquibase.sqlgenerator.core.ReorganizeTableGeneratorDB2
[2022-08-18 16:41:26] FINE [liquibase.servicelocator] Loaded liquibase.sqlgenerator.SqlGenerator instance liquibase.sqlgenerator.core.RuntimeGenerator
[2022-08-18 16:41:26] FINE [liquibase.servicelocator] Loaded liquibase.sqlgenerator.SqlGenerator instance liquibase.sqlgenerator.core.SelectFromDatabaseChangeLogGenerator
[2022-08-18 16:41:26] FINE [liquibase.servicelocator] Loaded liquibase.sqlgenerator.SqlGenerator instance liquibase.sqlgenerator.core.SelectFromDatabaseChangeLogLockGenerator
[2022-08-18 16:41:26] FINE [liquibase.servicelocator] Loaded liquibase.sqlgenerator.SqlGenerator instance liquibase.sqlgenerator.core.SetColumnRemarksGenerator
[2022-08-18 16:41:26] FINE [liquibase.servicelocator] Loaded liquibase.sqlgenerator.SqlGenerator instance liquibase.sqlgenerator.core.SetNullableGenerator
[2022-08-18 16:41:26] FINE [liquibase.servicelocator] Loaded liquibase.sqlgenerator.SqlGenerator instance liquibase.sqlgenerator.core.SetTableRemarksGenerator
[2022-08-18 16:41:26] FINE [liquibase.servicelocator] Loaded liquibase.sqlgenerator.SqlGenerator instance liquibase.sqlgenerator.core.StoredProcedureGenerator
[2022-08-18 16:41:26] FINE [liquibase.servicelocator] Loaded liquibase.sqlgenerator.SqlGenerator instance liquibase.sqlgenerator.core.TableRowCountGenerator
[2022-08-18 16:41:26] FINE [liquibase.servicelocator] Loaded liquibase.sqlgenerator.SqlGenerator instance liquibase.sqlgenerator.core.TagDatabaseGenerator
[2022-08-18 16:41:26] FINE [liquibase.servicelocator] Loaded liquibase.sqlgenerator.SqlGenerator instance liquibase.sqlgenerator.core.UnlockDatabaseChangeLogGenerator
[2022-08-18 16:41:26] FINE [liquibase.servicelocator] Loaded liquibase.sqlgenerator.SqlGenerator instance liquibase.sqlgenerator.core.UpdateChangeSetChecksumGenerator
[2022-08-18 16:41:26] FINE [liquibase.servicelocator] Loaded liquibase.sqlgenerator.SqlGenerator instance liquibase.sqlgenerator.core.UpdateDataChangeGenerator
[2022-08-18 16:41:26] FINE [liquibase.servicelocator] Loaded liquibase.sqlgenerator.SqlGenerator instance liquibase.sqlgenerator.core.UpdateGenerator
[2022-08-18 16:41:26] FINE [liquibase.servicelocator] Loaded liquibase.sqlgenerator.SqlGenerator instance liquibase.sqlgenerator.core.CreateSequenceGeneratorSnowflake
[2022-08-18 16:41:26] FINE [liquibase.servicelocator] Loaded liquibase.sqlgenerator.SqlGenerator instance liquibase.sqlgenerator.core.DropDefaultValueGeneratorSnowflake
[2022-08-18 16:41:26] FINE [liquibase.servicelocator] Loaded liquibase.sqlgenerator.SqlGenerator instance liquibase.sqlgenerator.core.DropProcedureGeneratorSnowflake
[2022-08-18 16:41:26] FINE [liquibase.servicelocator] Loaded liquibase.sqlgenerator.SqlGenerator instance liquibase.sqlgenerator.core.InsertOrUpdateGeneratorSnowflake
[2022-08-18 16:41:26] FINE [liquibase.servicelocator] Loaded liquibase.sqlgenerator.SqlGenerator instance liquibase.sqlgenerator.core.RenameTableGeneratorSnowflake
[2022-08-18 16:41:26] FINE [liquibase.servicelocator] Loaded liquibase.sqlgenerator.SqlGenerator instance liquibase.sqlgenerator.core.RenameViewGeneratorSnowflake
[2022-08-18 16:41:26] FINE [liquibase.servicelocator] Loaded liquibase.sqlgenerator.SqlGenerator instance liquibase.sqlgenerator.core.SetColumnRemarksGeneratorSnowflake
[2022-08-18 16:41:26] FINE [liquibase.servicelocator] Loaded liquibase.sqlgenerator.SqlGenerator instance liquibase.sqlgenerator.core.SetTableRemarksGeneratorSnowflake
[2022-08-18 16:41:26] FINE [liquibase.servicelocator] Loaded liquibase.sqlgenerator.SqlGenerator instance com.datical.liquibase.ext.appdba.markunused.change.MarkUnusedGenerator
[2022-08-18 16:41:26] FINE [liquibase.servicelocator] Loaded liquibase.sqlgenerator.SqlGenerator instance com.datical.liquibase.ext.appdba.synonym.change.CreateSynonymGenerator
[2022-08-18 16:41:26] FINE [liquibase.servicelocator] Loaded liquibase.sqlgenerator.SqlGenerator instance com.datical.liquibase.ext.appdba.synonym.change.DropSynonymGenerator
[2022-08-18 16:41:26] FINE [liquibase.servicelocator] Loaded liquibase.sqlgenerator.SqlGenerator instance com.datical.liquibase.ext.storedlogic.checkconstraint.change.AddCheckConstraintGenerator
[2022-08-18 16:41:26] FINE [liquibase.servicelocator] Loaded liquibase.sqlgenerator.SqlGenerator instance com.datical.liquibase.ext.storedlogic.checkconstraint.change.DisableCheckConstraintGenerator
[2022-08-18 16:41:26] FINE [liquibase.servicelocator] Loaded liquibase.sqlgenerator.SqlGenerator instance com.datical.liquibase.ext.storedlogic.checkconstraint.change.DropCheckConstraintGenerator
[2022-08-18 16:41:26] FINE [liquibase.servicelocator] Loaded liquibase.sqlgenerator.SqlGenerator instance com.datical.liquibase.ext.storedlogic.checkconstraint.change.EnableCheckConstraintGenerator
[2022-08-18 16:41:26] FINE [liquibase.servicelocator] Loaded liquibase.sqlgenerator.SqlGenerator instance com.datical.liquibase.ext.storedlogic.databasepackage.change.CreatePackageBodyGenerator
[2022-08-18 16:41:26] FINE [liquibase.servicelocator] Loaded liquibase.sqlgenerator.SqlGenerator instance com.datical.liquibase.ext.storedlogic.databasepackage.change.CreatePackageGenerator
[2022-08-18 16:41:26] FINE [liquibase.servicelocator] Loaded liquibase.sqlgenerator.SqlGenerator instance com.datical.liquibase.ext.storedlogic.databasepackage.change.DropPackageBodyGenerator
[2022-08-18 16:41:26] FINE [liquibase.servicelocator] Loaded liquibase.sqlgenerator.SqlGenerator instance com.datical.liquibase.ext.storedlogic.databasepackage.change.DropPackageGenerator
[2022-08-18 16:41:26] FINE [liquibase.servicelocator] Loaded liquibase.sqlgenerator.SqlGenerator instance com.datical.liquibase.ext.storedlogic.function.change.CreateFunctionGenerator
[2022-08-18 16:41:26] FINE [liquibase.servicelocator] Loaded liquibase.sqlgenerator.SqlGenerator instance com.datical.liquibase.ext.storedlogic.function.change.DropFunctionGenerator
[2022-08-18 16:41:26] FINE [liquibase.servicelocator] Loaded liquibase.sqlgenerator.SqlGenerator instance com.datical.liquibase.ext.storedlogic.trigger.change.CreateTriggerGenerator
[2022-08-18 16:41:26] FINE [liquibase.servicelocator] Loaded liquibase.sqlgenerator.SqlGenerator instance com.datical.liquibase.ext.storedlogic.trigger.change.DisableTriggerGenerator
[2022-08-18 16:41:26] FINE [liquibase.servicelocator] Loaded liquibase.sqlgenerator.SqlGenerator instance com.datical.liquibase.ext.storedlogic.trigger.change.DropTriggerGenerator
[2022-08-18 16:41:26] FINE [liquibase.servicelocator] Loaded liquibase.sqlgenerator.SqlGenerator instance com.datical.liquibase.ext.storedlogic.trigger.change.EnableTriggerGenerator
[2022-08-18 16:41:26] FINE [liquibase.servicelocator] Loaded liquibase.sqlgenerator.SqlGenerator instance com.datical.liquibase.ext.storedlogic.trigger.change.RenameTriggerGenerator
[2022-08-18 16:41:26] FINE [liquibase.executor] Executing with the 'jdbc' executor
[2022-08-18 16:41:26] FINE [liquibase.configuration] No configuration value for liquibase.sql.logLevel found
[2022-08-18 16:41:26] FINE [liquibase.configuration] Configuration liquibase.sql.logLevel is using the default value of FINE
[2022-08-18 16:41:26] FINE [liquibase.executor] USE null
[2022-08-18 16:41:26] SEVERE [liquibase.integration] Incorrect syntax near the keyword 'null'. [Failed SQL: (156) USE null]
liquibase.exception.CommandExecutionException: liquibase.exception.DatabaseException: liquibase.exception.DatabaseException: Incorrect syntax near the keyword 'null'. [Failed SQL: (156) USE null]
        at liquibase.command.CommandScope.execute(CommandScope.java:179)
        at liquibase.integration.commandline.CommandRunner.call(CommandRunner.java:51)
        at liquibase.integration.commandline.CommandRunner.call(CommandRunner.java:21)
        at picocli.CommandLine.executeUserObject(CommandLine.java:1953)
        at picocli.CommandLine.access$1300(CommandLine.java:145)
        at picocli.CommandLine$RunLast.executeUserObjectOfLastSubcommandWithSameParent(CommandLine.java:2358)
        at picocli.CommandLine$RunLast.handle(CommandLine.java:2352)
        at picocli.CommandLine$RunLast.handle(CommandLine.java:2314)
        at picocli.CommandLine$AbstractParseResultHandler.execute(CommandLine.java:2179)
        at picocli.CommandLine$RunLast.execute(CommandLine.java:2316)
        at picocli.CommandLine.execute(CommandLine.java:2078)
        at liquibase.integration.commandline.LiquibaseCommandLine.lambda$execute$1(LiquibaseCommandLine.java:353)
        at liquibase.Scope.child(Scope.java:189)
        at liquibase.Scope.child(Scope.java:165)
        at liquibase.integration.commandline.LiquibaseCommandLine.execute(LiquibaseCommandLine.java:318)
        at liquibase.integration.commandline.LiquibaseCommandLine.main(LiquibaseCommandLine.java:84)
        at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
        at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(Unknown Source)
        at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(Unknown Source)
        at java.base/java.lang.reflect.Method.invoke(Unknown Source)
        at liquibase.integration.commandline.LiquibaseLauncher.main(LiquibaseLauncher.java:107)
Caused by: liquibase.exception.DatabaseException: liquibase.exception.DatabaseException: Incorrect syntax near the keyword 'null'. [Failed SQL: (156) USE null]
        at liquibase.integration.commandline.CommandLineUtils.createDatabaseObject(CommandLineUtils.java:139)
        at liquibase.integration.commandline.Main.doMigration(Main.java:1485)
        at liquibase.integration.commandline.Main$1.lambda$run$0(Main.java:397)
        at liquibase.Scope.lambda$child$0(Scope.java:180)
        at liquibase.Scope.child(Scope.java:189)
        at liquibase.Scope.child(Scope.java:179)
        at liquibase.Scope.child(Scope.java:158)
        at liquibase.integration.commandline.Main$1.run(Main.java:396)
        at liquibase.integration.commandline.Main$1.run(Main.java:218)
        at liquibase.Scope.child(Scope.java:189)
        at liquibase.Scope.child(Scope.java:165)
        at liquibase.integration.commandline.Main.run(Main.java:218)
        at liquibase.command.AbstractCliWrapperCommandStep.run(AbstractCliWrapperCommandStep.java:33)
        at liquibase.command.CommandScope.execute(CommandScope.java:173)
        ... 20 more
Caused by: liquibase.exception.DatabaseException: Incorrect syntax near the keyword 'null'. [Failed SQL: (156) USE null]
        at liquibase.executor.jvm.JdbcExecutor$ExecuteStatementCallback.doInStatement(JdbcExecutor.java:440)
        at liquibase.executor.jvm.JdbcExecutor.execute(JdbcExecutor.java:78)
        at liquibase.executor.jvm.JdbcExecutor.execute(JdbcExecutor.java:161)
        at liquibase.executor.jvm.JdbcExecutor.execute(JdbcExecutor.java:126)
        at liquibase.database.core.DatabaseUtils.initializeDatabase(DatabaseUtils.java:76)
        at liquibase.integration.commandline.CommandLineUtils.createDatabaseObject(CommandLineUtils.java:135)
        ... 33 more
Caused by: com.microsoft.sqlserver.jdbc.SQLServerException: Incorrect syntax near the keyword 'null'.
        at com.microsoft.sqlserver.jdbc.SQLServerException.makeFromDatabaseError(SQLServerException.java:265)
        at com.microsoft.sqlserver.jdbc.SQLServerStatement.getNextResult(SQLServerStatement.java:1676)
        at com.microsoft.sqlserver.jdbc.SQLServerStatement.doExecuteStatement(SQLServerStatement.java:907)
        at com.microsoft.sqlserver.jdbc.SQLServerStatement$StmtExecCmd.doExecute(SQLServerStatement.java:802)
        at com.microsoft.sqlserver.jdbc.TDSCommand.execute(IOBuffer.java:7730)
        at com.microsoft.sqlserver.jdbc.SQLServerConnection.executeCommand(SQLServerConnection.java:3786)
        at com.microsoft.sqlserver.jdbc.SQLServerStatement.executeCommand(SQLServerStatement.java:268)
        at com.microsoft.sqlserver.jdbc.SQLServerStatement.executeStatement(SQLServerStatement.java:242)
        at com.microsoft.sqlserver.jdbc.SQLServerStatement.execute(SQLServerStatement.java:775)
        at liquibase.executor.jvm.JdbcExecutor$ExecuteStatementCallback.doInStatement(JdbcExecutor.java:436)
        ... 38 more


Unexpected error running Liquibase: Incorrect syntax near the keyword 'null'. [Failed SQL: (156) USE null]

liquibase.exception.CommandExecutionException: liquibase.exception.DatabaseException: liquibase.exception.DatabaseException: Incorrect syntax near the keyword 'null'. [Failed SQL: (156) USE null]
        at liquibase.command.CommandScope.execute(CommandScope.java:179)
        at liquibase.integration.commandline.CommandRunner.call(CommandRunner.java:51)
        at liquibase.integration.commandline.CommandRunner.call(CommandRunner.java:21)
        at picocli.CommandLine.executeUserObject(CommandLine.java:1953)
        at picocli.CommandLine.access$1300(CommandLine.java:145)
        at picocli.CommandLine$RunLast.executeUserObjectOfLastSubcommandWithSameParent(CommandLine.java:2358)
        at picocli.CommandLine$RunLast.handle(CommandLine.java:2352)
        at picocli.CommandLine$RunLast.handle(CommandLine.java:2314)
        at picocli.CommandLine$AbstractParseResultHandler.execute(CommandLine.java:2179)
        at picocli.CommandLine$RunLast.execute(CommandLine.java:2316)
        at picocli.CommandLine.execute(CommandLine.java:2078)
        at liquibase.integration.commandline.LiquibaseCommandLine.lambda$execute$1(LiquibaseCommandLine.java:353)
        at liquibase.Scope.child(Scope.java:189)
        at liquibase.Scope.child(Scope.java:165)
        at liquibase.integration.commandline.LiquibaseCommandLine.execute(LiquibaseCommandLine.java:318)
        at liquibase.integration.commandline.LiquibaseCommandLine.main(LiquibaseCommandLine.java:84)
        at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
        at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(Unknown Source)
        at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(Unknown Source)
        at java.base/java.lang.reflect.Method.invoke(Unknown Source)
        at liquibase.integration.commandline.LiquibaseLauncher.main(LiquibaseLauncher.java:107)
Caused by: liquibase.exception.DatabaseException: liquibase.exception.DatabaseException: Incorrect syntax near the keyword 'null'. [Failed SQL: (156) USE null]
        at liquibase.integration.commandline.CommandLineUtils.createDatabaseObject(CommandLineUtils.java:139)
        at liquibase.integration.commandline.Main.doMigration(Main.java:1485)
        at liquibase.integration.commandline.Main$1.lambda$run$0(Main.java:397)
        at liquibase.Scope.lambda$child$0(Scope.java:180)
        at liquibase.Scope.child(Scope.java:189)
        at liquibase.Scope.child(Scope.java:179)
        at liquibase.Scope.child(Scope.java:158)
        at liquibase.integration.commandline.Main$1.run(Main.java:396)
        at liquibase.integration.commandline.Main$1.run(Main.java:218)
        at liquibase.Scope.child(Scope.java:189)
        at liquibase.Scope.child(Scope.java:165)
        at liquibase.integration.commandline.Main.run(Main.java:218)
        at liquibase.command.AbstractCliWrapperCommandStep.run(AbstractCliWrapperCommandStep.java:33)
        at liquibase.command.CommandScope.execute(CommandScope.java:173)
        ... 20 more
Caused by: liquibase.exception.DatabaseException: Incorrect syntax near the keyword 'null'. [Failed SQL: (156) USE null]
        at liquibase.executor.jvm.JdbcExecutor$ExecuteStatementCallback.doInStatement(JdbcExecutor.java:440)
        at liquibase.executor.jvm.JdbcExecutor.execute(JdbcExecutor.java:78)
        at liquibase.executor.jvm.JdbcExecutor.execute(JdbcExecutor.java:161)
        at liquibase.executor.jvm.JdbcExecutor.execute(JdbcExecutor.java:126)
        at liquibase.database.core.DatabaseUtils.initializeDatabase(DatabaseUtils.java:76)
        at liquibase.integration.commandline.CommandLineUtils.createDatabaseObject(CommandLineUtils.java:135)
        ... 33 more
Caused by: com.microsoft.sqlserver.jdbc.SQLServerException: Incorrect syntax near the keyword 'null'.
        at com.microsoft.sqlserver.jdbc.SQLServerException.makeFromDatabaseError(SQLServerException.java:265)
        at com.microsoft.sqlserver.jdbc.SQLServerStatement.getNextResult(SQLServerStatement.java:1676)
        at com.microsoft.sqlserver.jdbc.SQLServerStatement.doExecuteStatement(SQLServerStatement.java:907)
        at com.microsoft.sqlserver.jdbc.SQLServerStatement$StmtExecCmd.doExecute(SQLServerStatement.java:802)
        at com.microsoft.sqlserver.jdbc.TDSCommand.execute(IOBuffer.java:7730)
        at com.microsoft.sqlserver.jdbc.SQLServerConnection.executeCommand(SQLServerConnection.java:3786)
        at com.microsoft.sqlserver.jdbc.SQLServerStatement.executeCommand(SQLServerStatement.java:268)
        at com.microsoft.sqlserver.jdbc.SQLServerStatement.executeStatement(SQLServerStatement.java:242)
        at com.microsoft.sqlserver.jdbc.SQLServerStatement.execute(SQLServerStatement.java:775)
        at liquibase.executor.jvm.JdbcExecutor$ExecuteStatementCallback.doInStatement(JdbcExecutor.java:436)
        ... 38 more
```
</details>

--- a/.gitignore
+++ b/.gitignore
@@ -35,3 +35,4 @@ original-pom.xml
 **/logback-test.xml
 **/*.swp
 **/dependency-reduced-pom.xml
+compareGeneratedSqlWithExpectedSqlForMinimalChangesets/

--- a/.gitignore
+++ b/.gitignore
@@ -35,4 +35,3 @@ original-pom.xml
 **/logback-test.xml
 **/*.swp
 **/dependency-reduced-pom.xml
-compareGeneratedSqlWithExpectedSqlForMinimalChangesets/

--- a/liquibase-core/src/main/java/liquibase/database/core/DatabaseUtils.java
+++ b/liquibase-core/src/main/java/liquibase/database/core/DatabaseUtils.java
@@ -73,9 +73,11 @@ public class DatabaseUtils {
                 }
                 executor.execute(new RawSqlStatement("USE " + schema));
             } else if (database instanceof MSSQLDatabase) {
-                executor.execute(new RawSqlStatement("USE " + defaultCatalogName));
+                    defaultCatalogName = StringUtil.trimToNull(defaultCatalogName);
+                    if (defaultCatalogName != null) {
+                        executor.execute(new RawSqlStatement(String.format("USE %s", defaultCatalogName)));
+                    }
             }
-
         }
     }
 

